src/consensus/aft/raft.h

```diff
@@ -595,12 +591,19 @@ namespace aft
                 *vc, id(), new_view, seqno, node_count()) &&
             get_primary(new_view) == id())
           {
+            // We need to reobtain the lock when writing to the ledger so we
+            // need to release it at this time.
```

Comment:
Just making sure I've understood this correctly: `append_new_view()` appends to the ledger in `write_view_change_confirmation()`, which writes to the ledger though KV transaction commit, then `replicate()`?

Reply:
yes